src/components/UserMenu.vue

```diff
@@ -3,7 +3,7 @@
     <div class="user-menu__header">
       <button class="user-menu__close-avatar" type="button" @click="sendCloseEvent()">
         <span class="visually-hidden">Close user menu</span>
-        <UserPicture :picture="picture.value" :username="userId.value" size="40"></UserPicture>
+        <UserPicture :picture="picture.value" :username="userId.value" :size="40"></UserPicture>
```

Comment:
wouldn't `size=40` work here, too?

Reply:
With `:size` we are able to pass it as a JS expression, so that it is sent down as a `Number`.